cairo/src/state.cairo

```diff
@@ -305,13 +278,11 @@ namespace State {
             accounts=state.accounts,
             events_len=state.events_len + 1,
             events=state.events,
-            transfers_len=state.transfers_len,
-            transfers=state.transfers,
         );
         return ();
     }
 
-    // @notice Add a transfer to the Transfer* array
+    // @notice Processes a transfer (we don't need to include the it in the state)
```

Comment:
```suggestion
    // @notice Process a transfer
    // @dev Check and update balances
```

the comment `(we don't need to include ...` is really based on a previous non relevant context (being on sn)